lib/stream.js

```diff
@@ -49,7 +49,8 @@ try {
   } else {
     // This throws for Node < 4.2.0 because there's no util binding and
     // returns undefined for Node < 7.4.0.
-    Stream._isUint8Array = process.binding('util').isUint8Array;
+    const { internalBinding } = require('internal/bootstrap/loaders');
+    Stream._isUint8Array = internalBinding('util').isUint8Array;
```

Comment:
I think we want to revert this change, since it is only left in the file for compatibility with prior Node.js versions in `readable-stream`.

/cc @nodejs/streams